server/database/connection/index.js

```diff
@@ -0,0 +1,13 @@
+require("env2")(".env");
+const { Pool } = require("pg");
+
+const connectionString = process.env.DB_URL;
```

Comment:
You already define NODE_ENV in your package.json to use it here : 
```js
const {
  NODE_ENV,
  DB_DEV_URL,
  DATABASE_URL,
  TEST_DB_URL,
} = process.env;
```

Then initialize the switch case to know which DB to use 
```js
switch (NODE_ENV) {
  case 'production':
    dbUrl = DATABASE_URL;
    break;
  case 'development':
    dbUrl = DB_DEV_URL;
    break;
  case 'test':
    dbUrl = TEST_DB_URL;
    break;
  default:
    throw new Error('NO DATABASE to show!');
}
```